draft-ietf-quic-transport.md

```diff
@@ -1853,7 +1853,9 @@ An endpoint also MUST NOT initiate connection migration if the peer sent the
 `disable_migration` transport parameter during the handshake.  An endpoint which
 has sent this transport parameter, but detects that a peer has nonetheless
 migrated to a different network MAY treat this as a connection error of type
-INVALID_MIGRATION.
+INVALID_MIGRATION.  Similarly, an endpoint MUST NOT migrate if its peer supplies
```

Comment:
As I said on the list, I'm not a big fan of adding this 0-len CID restriction.

Reply:
Not adding; linking to #2441 where this separate issue will be discussed.